contrib/storage-kafka/src/test/java/org/apache/drill/exec/store/kafka/KafkaFilterPushdownTest.java

```diff
@@ -63,7 +62,9 @@ public void testPushdownOnOffset() throws Exception {
         TestQueryConstants.JSON_PUSHDOWN_TOPIC, predicate1, predicate2);
 
     runKafkaSQLVerifyCount(queryString, expectedRowCount);
-    testPhysicalPlan(queryString, String.format(expectedSubStr, expectedRowCount));
+    PlanTestBase.testPlanWithAttributesMatchingPatterns(queryString,
```

Comment:
also since you moved to using `testPlanWithAttributesMatchingPatterns` from `testPhysicalPlan` now the `rowCount` value is different than `expectedRowCount`. Is that intended ?